articles/architecture-scenarios/application/mobile-api.md

```diff
@@ -4,35 +4,252 @@ title: Mobile + API
+
+<%= include('../_includes/_api-configure-scopes.md') %>
+
+### Create the Client
+
+There are four client types in Auth0: __Native__ (used by mobile or desktop apps), __Single Page Web Applications__, __Regular Web Applications__ and __Non Interactive Clients__ (used by CLIs, Daemons, or services running on your backend). For this scenario we want to create a new Client for our Mobile Application, hence we will use Native as the client type.
+
+To create a new Client, navigate to the [dashboard](${manage_url}) and click on the [Clients](${manage_url}/#/clients}) menu option on the left. Click the __+ Create Client__ button.
+
+Set a name for your Client (we will use `Timesheets Mobile`) and select `Native` as the type.
+
+Click __Create__.
+
+![Create Client](/media/articles/architecture-scenarios/mobile-api/create-client.png)
+
+That's it for now. When we are done with the Mobile app implementation we will revisit the dashboard and this Client's settings to make some changes in its configuration.
+
+## Inside the Implementation
+
+<%= include('../_includes/_api-implement.md') %>
+
+### Implement the Mobile App
+
+In this section we will see how we can implement a Mobile application for our scenario.
+
 ::: note
-This architecture scenario is under construction and will be updated soon.
+[See the implementation in Android.](/architecture-scenarios/application/mobile-api/mobile-implementation-android#1-set-up-the-application)
 :::
 
-![Mobile + API Flow](/media/articles/architecture-scenarios/mobile-api.png)
+#### Authorize the User
 
-In this scenario you have a mobile application ("Client") which talks to an API ("Resource Server"). The application will use **OpenID Connect** with the **Authorization Code Grant using Proof Key for Code Exchange (PKCE)** to authenticate users. Note that this flow can only be used for Clients whose type is **Native** in the [Dashboard](${manage_url}).
+To authorize the user we will implement an [Authorization Code Grant Flow with PKCE](/api-auth/tutorials/authorization-code-grant-pkce). The mobile application should first send the user to the [authorization URL](/api/authentication#authorization-code-grant-pkce-) along with the `code_challenge` and the method used to generate it:
 
-When a user logs in, Auth0 will return to the application an `access_token`, an `id_token`, and optionally a `refresh_token`:
+```text
+https://${account.namespace}/authorize?
+    audience=API_AUDIENCE&
+    scope=SCOPE&
+    response_type=code&
+    client_id=YOUR_CLIENT_ID&
+    code_challenge=CODE_CHALLENGE&
+    code_challenge_method=S256&
+    redirect_uri=https://YOUR_APP/callback
+```
 
-- The `access_token` is used to securely call the API on behalf of the user.
+The `GET` request to the authorization URL should include the following values:
+
+- __domain__: The value of your Auth0 Domain. You can retrieve it from the Settings of your Client at the [Auth0 Dashboard](${manage_url}/#/clients).
```

Comment:
Can you change this from a list of bullets to a table format? I think it will look better

| __Parameter__ | __Description__ |
| __domain__ | The value of your Auth0 Domain. You can retrieve it from the Settings of your Client at the [Auth0 Dashboard](${manage_url}/#/clients).
| __client_id__ | The value of your Auth0 Client Id. You can retrieve it from the Settings of your Client at the [Auth0 Dashboard](${manage_url}/#/clients).

There are two more snippets, that have their params listed, in this doc, change them as well.